src/main/java/com/google/api/codegen/util/go/GoTypeTable.java

```diff
@@ -104,4 +105,38 @@ private static boolean isStandardImport(String importPath) {
     // we might have to special case them.
     return !importPath.contains("/");
   }
+
+  /**
+   * A set of Go reserved words. See
+   * https://golang.org/ref/spec#Keywords
+   */
+  public static final ImmutableSet<String> KEYWORD_BUILT_IN_SET =
```

Comment:
Though `RESERVED_IDENTIFIER_SET` is a more accurate name if we do decide to include the names in builtin:

https://golang.org/pkg/builtin/

All of these can be overridden by declaring a var of the same name